### PR TITLE
Adds a Updated at to the Install Panel

### PR DESCRIPTION
**Pull request by TerminalFi**
Adds a Updated at to the Install Panel

image

### CODE REVIEW

File: package_control/package_tasks.py [hunk 1]
```diff
@@ -624,7 +631,11 @@ def render_quick_panel_items(self, tasks):
                         final_line += ' '
                     final_line += '<a href="{}">{}</a>'.format(homepage, homepage_display)
 
-                items.append(sublime.QuickPanelItem(task.package_name, [description, final_line]))
+                annotation = ''
+                if task.last_modified:
+                    annotation = datetime.datetime.strptime(task.last_modified, '%Y-%m-%d %H:%M:%S').strftime('Updated at %a %b %d %H:%M:%S %Y')
```
Comment (rchl) [inline package_control/package_tasks.py:636]: I would think that it's excessive with time. Day, month and year would probably be enough?

File: package_control/package_tasks.py [hunk 2]
```diff
@@ -624,7 +631,14 @@ def render_quick_panel_items(self, tasks):
                         final_line += ' '
                     final_line += '<a href="{}">{}</a>'.format(homepage, homepage_display)
 
-                items.append(sublime.QuickPanelItem(task.package_name, [description, final_line]))
+                annotation = ''
+                if task.last_modified:
+                    annotation = datetime.strptime(
+                        task.last_modified,
+                        '%Y-%m-%d %H:%M:%S'
+                    ).strftime('Updated at %a %b %d, %Y')
```
[the remark this answers is not in the document]
Comment (TerminalFi) [reply package_control/package_tasks.py:639]: %c reintroduces the time. Which based on @rchl comment makes sense to remove. Thoughts on shifting `existing_packages` to `%a %b %d, %Y'`

[the remark this answers is not in the document]
Comment (TerminalFi) [reply package_control/package_tasks.py:639]: Or even using `%x`

[the remark this answers is not in the document]
Comment (FichteFoll) [reply package_control/package_tasks.py:639]: If the time is supposed to be omitted, "updated on" sounds more natural.

[the remark this answers is not in the document]
Comment (deathaxe) [reply package_control/package_tasks.py:639]: Fixed on four-point-oh

**Pull request merged**